packages/http-client-csharp/package-lock.json

```diff
@@ -45,8 +45,6 @@
     },
     "node_modules/@ampproject/remapping": {
       "version": "2.3.0",
-      "resolved": "https://registry.npmjs.org/@ampproject/remapping/-/remapping-2.3.0.tgz",
```

Comment:
There are a bunch of unrelated changes - if we run the command I mentioned in the other comment we will only have the version change.